xsuite/src/proxy/csproxy.ts

```diff
@@ -0,0 +1,131 @@
+import { addressLikeToBechAddress } from "../data/addressLike";
+import { e, eCodeMetadata, EncodableAccount } from "../data/encoding";
+import { Kvs } from "../data/kvs";
+import { BroadTx, Proxy } from "./proxy";
+
+export class CSProxy extends Proxy {
+  autoGenerateBlocks: boolean;
+  waitCompletedTimeout: number;
+
+  constructor(params: CSProxyParams) {
+    super(params);
+
+    this.autoGenerateBlocks = params.autoGenerateBlocks;
+    this.waitCompletedTimeout = params.waitCompletedTimeout ?? 250;
```

Comment:
Let's call it `txCompletionPauseMs`. This param should be introduced at `Proxy` level. And in `getCompletedTx` of `Proxy`, this param should be used instead of 1000.

Reply:
Done

Reply:
Not done. How is it possible you could say it is done?

Reply:
Properly done now